spring-integration-file/src/main/java/org/springframework/integration/file/remote/synchronizer/AbstractInboundFileSynchronizer.java

```diff
@@ -227,6 +228,11 @@ public void close() throws IOException {
 
 	@Override
 	public void synchronizeToLocalDirectory(final File localDirectory) {
+		synchronizeToLocalDirectory(localDirectory, Integer.MIN_VALUE);
```

Comment:
Somehow it looks inconsistent with the interface contract.
I was thinking about to change the `default` method in the interface, but would it be correct then to require from each implementors to deal with `maxFetchSize` ?.. 
Just a loud thought. 
But I don't mind to discuss the concern a bit.
Although I may live with this solution as well.

I have some hard work in SI-AWS S3 `MessageSource` to push `maxFetchSize` to the AmazonS3 directly somehow:
```
  * @param maxKeys
     *            The maximum number of results to return.
     *
     * @see ListObjectsRequest#ListObjectsRequest()
     */
    public ListObjectsRequest(String bucketName, String prefix, String marker, String delimiter, Integer maxKeys) {
```